src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -109,23 +113,16 @@ def save_pretrained_wrapper(
             if compressor is None:
                 # model is not compressed or quantized, save as normal
                 original_save_pretrained.__get__(model, model_class)(
-                    save_directory, **kwargs
+                    save_directory, state_dict=state_dict, **kwargs
                 )
                 return
 
-            # if we've gotten to this point we have a config so we can run compression
-            # default safe serialization to True if not explicitly set
-            kwargs["safe_serialization"] = kwargs.get("safe_serialization", True)
```

Comment:
This is already True by default?

Reply:
Yes, but let's make it explicit

Reply:
[Yes](https://huggingface.co/docs/transformers/v4.44.2/en/main_classes/model#transformers.PreTrainedModel.save_pretrained), although there's no harm in keeping this line.